.copr/Makefile

```diff
@@ -1,8 +1,22 @@
 .PHONY: installdeps git_cfg_safe srpm
 
 installdeps:
-	dnf -y install ant git httpcomponents-client jackson-annotations jackson-core jackson-databind java-11-openjdk-devel javapackages-local \
-		libxslt maven maven-local python3 python3-lxml rpm-build rpmdevtools slf4j
+	dnf -y install \
+		ant \
+		git \
+		httpcomponents-client \
+		jackson-annotations \
+		jackson-core \
+		jackson-databind \
+		java-21-openjdk-devel \
+		libxslt \
+		maven \
+		javapackages-local \
```

Comment:
Think you better depend on maven-local-openjdk21 (which will also install maven etc then)